.travis/deploy_npm

```diff
@@ -40,6 +40,8 @@ exe npx qx --version
 exe npm update --no-save @qooxdoo/compiler 
 exe npx qx --version
 exe npx qx compile -v
+exe mkdir source
```

Comment:
Is `source` guaranteed not to exist? If not, this should be `exe mkdir -p source` so that it creates it if it doesn't eixist,and doesn't barf if it does like the current could would do.

Reply:
@hkollmann what do you think ? I guess since this runs only on travis, it should be "clean"...